documentation/decisions/007-distinct-references-section-for-LGs.md

```diff
@@ -0,0 +1,17 @@
+# 007: Distinct References Section Per LG
+
+## Status: Accepted
+
+## Context 
+We have a section for references per learning goal.
+
+## Rationale:
+
+In previous versions of the curriculum we kept references (books, websites) per section of the curriculum.
+It was difficult (if not impossible) to find out which of these references was useful for which part of the content.
```

Comment:
I would write it more explicitly here: 

... was useful for which learning goal.

Reply:
done